Sources/Clickstream/NetworkManager/Core/RetryMechanism.swift

```diff
@@ -58,6 +62,24 @@ final class DefaultRetryMechanism: Retryable {
             Tracker.sharedInstance?.record(event: healthEvent)
         }
         
+        if !isConnected && Tracker.debugMode {
```

Comment:
HealthTracking should be wrapped in `#if TRACKER_ENABLED`, else it might give error

Reply:
Line 65 and line 66 is same